src/main/java/rx/internal/util/BackpressureDrainManager.java

```diff
@@ -138,7 +135,7 @@ public final void request(long n) {
         long r;
         long u;
         do {
-            r = requestedCount;
+            r = requestedCount.get();
```

Comment:
If not inlined, `requestedCount` reference should be read into a local variable outside the loop.